ws-nextjs-app/pages/[service]/send/[id]/FormContext/utils/getValidationErrors.ts

```diff
@@ -0,0 +1,20 @@
+import { FieldData, OnChangeInputName, ValidationError } from '../../types';
+
+const getValidationErrors = (state: Record<OnChangeInputName, FieldData>) => {
+  const formEntriesNew = new Map(Object.entries(state));
+
+  const errorList: ValidationError[] = [];
+
+  formEntriesNew.forEach((data, key) => {
+    const { isValid, messageCode } = data;
+    if (isValid === false) {
+      errorList.push({
+        id: key,
+        messageCode,
```

Comment:
Heya, was just wondering, since we just display a simpler error message, do we even need this one anymore?